src/protocol/templates/pingmessage.py.in

```diff
@@ -370,31 +376,32 @@ if __name__ == "__main__":
 
     # Hand-written data buffers for testing and verification
     test_voltage_5_buf = bytearray([0x42, 0x52, 0x02, 0x00, 0xB2, 0x04, 0x05, 0x06, 0x01, 0x01, 0x59, 0x01])
-    test_nack_buf = bytearray([0x42, 0x52, 0x06, 0x00, 0x03, 0x00, 0x05, 0x06, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0xBC, 0x02])
-    test_profile_buf = bytearray([0x42, 0x52,  0x1E, 0x00,  0x14, 0x05,  0x05, 0x06,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f,
-                               0x1, 0x2, 0x63, 0x64,
-                               0xB2, 0x0A])
+    test_nack_buf = bytearray(
+        [0x42, 0x52, 0x06, 0x00, 0x03, 0x00, 0x05, 0x06, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0xBC, 0x02])
+    test_profile_buf = bytearray([0x42, 0x52, 0x1E, 0x00, 0x14, 0x05, 0x05, 0x06,
```

Comment:
what happened here? removed newlines?

Reply:
just reduced line length and made aligned indentation